core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -98,6 +98,11 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val METRICS_PROCESS_TREE_METRICS =
+    ConfigBuilder("spark.metrics.logStageExecutorProcessTreeMetrics.enabled")
```

Comment:
Shouldn't the name of config be different? Something like: `spark.metrics.addProcessTreeMetricsToMetricsSet.enabled`

Reply:
Yeah, I will change that. Thanks.